client/jetpack-onboarding/steps/summary.jsx

```diff
@@ -108,9 +116,22 @@ export default connect( ( state, { siteId, steps } ) => {
 	const tasks = compact( [] );
 	const stepsCompleted = getJetpackOnboardingProgress( state, siteId, steps );
 
+	const stepActionsMap = {
+		[ STEPS.WOOCOMMERCE ]: {
+			installWooCommerce: true,
+		},
+	};
+
+	const stepsPending = mapValues(
+		stepActionsMap,
+		// eslint-disable-next-line wpcalypso/redux-no-bound-selectors
+		action => getRequest( state, saveJetpackOnboardingSettings( siteId, action ) ).isLoading
```

Comment:
Can we avoid creating this arrow function on every rerender? Perhaps it makes sense to introduce a selector for pending steps (or extend `getJetpackOnboardingProgress`)? Would be very useful if we decide to implement it for each step (which I think would make sense for slow connections and slowly responding Jetpack sites). Can be done in a subsequent PR of course.

Reply:
Makes sense. I'll go for a separate PR 😉

Reply:
#21565